api/Document.json

```diff
@@ -8171,16 +8179,22 @@
           "support": {
             "chrome": [
               {
-                "version_added": "33"
+                "version_added": "62"
               },
               {
                 "version_added": "13",
                 "prefix": "webkit"
               }
             ],
-            "chrome_android": {
-              "version_added": "33"
-            },
+            "chrome_android": [
+              {
+                "version_added": "62"
+              },
+              {
+                "version_added": "18",
+                "prefix": "webkit"
```

Comment:
I now see the data being mirrored here is wrong. `onwebkitvisibilitychange` isn't in current Chrome. Can you add a custom test for it in the collector and confirm `version_added` + add right `version_removed`?

I also note that it's not a prefix, since the name of this isn't `webkitonvisibilitychange`, so switching to `alternative_name` would be good. But maybe as a part of a mass change PR for all `onfoo` event handler attributes involving prefixes.

Reply:
It turns out that `onwebkitvisibilitychange` wasn't supported in Chrome; testing in Chrome 15 revealed no support.  The original data came from wiki mirroring, so I don't trust the data much.  Reverting the mirroring and removing the data.

Reply:
Yay, thanks!